freesound/middleware.py

```diff
@@ -119,3 +119,28 @@ def __call__(self, request):
 
         response = self.get_response(request)
         return response
+
+
+class UpdateEmailHandler(object):
+    def __init__(self, get_response):
+        self.get_response = get_response
+
+    def __call__(self, request):
+        if request.user.is_authenticated \
```

Comment:
In gdpr branch we added a util function to handle the "don't redirect" URLs. When eventually merging gdpr to master and master to this branch we should update that function to include `resetemail` and use it here.

Reply:
Yes, as I mentioned in the comment below :)